src/shared/components/LomFieldsInput/LomFieldsInput.tsx

```diff
@@ -41,41 +42,61 @@ const LomFieldsInput: FC<LomFieldsInputProps> = ({
 	const { data: allSubjects, isLoading: isSubjectsLoading } = useGetLomFields('subject');
 	const { data: allThemes, isLoading: isThemesLoading } = useGetLomFields('theme');
 
+	const getEducationLevelsOptions = (allEducationLevels: Avo.Lom.LomField[]) => {
+		const degrees = filterAllEduLevels(allEducationLevels);
+		const parentIdsOfDegrees = map(degrees, 'broader');
+		const levels = filter(allEducationLevels, (level) => isNil(level.broader));
```

Comment:
We can use the groupLoms function for this, instead of checking the broader is null
that way we keep the logic for identifying educationLevels from educationDegrees all in one location